server/test/mdm.go

```diff
@@ -36,6 +36,14 @@ func CreateVPPTokenEncoded(expiration time.Time, orgName, location string) ([]by
 	if err != nil {
 		return nil, err
 	}
+	return []byte(dataToken.Token), nil
+}
+
+func CreateVPPTokenEncodedAfterMigration(expiration time.Time, orgName, location string) ([]byte, error) {
```

Comment:
The format returned by this is only valid/expected in the DB after coming from the DB migration for an existing VPP token.